cirq/google/sim/xmon_simulator_test.py

```diff
@@ -61,6 +61,12 @@ def large_circuit():
     return circuit
 
 
+def empty_measurement_circuit(qubit):
```

Comment:
You use this method once, and it can be compressed into a single expression `cirq.Circuit.from_ops(cirq.measure(qubit, key='meas'))`. Inline it.